openfe/tests/protocols/test_openmm_equil_rfe_protocols.py

```diff
@@ -1241,10 +1241,101 @@ def test_constraints(tyk2_xml, tyk2_reference_xml):
             assert float(a.get('d')) == pytest.approx(float(b.get('d')))
 
 
-def test_reload_protocol_result(transformation_json):
-    d = json.loads(transformation_json,
-                   cls=gufe.tokenization.JSON_HANDLER.decoder)
+class TestProtocolResult:
+    @pytest.fixture()
+    def protocolresult(self, transformation_json):
+        d = json.loads(transformation_json,
+                       cls=gufe.tokenization.JSON_HANDLER.decoder)
 
-    pr = openmm_rfe.RelativeHybridTopologyProtocolResult.from_dict(d['protocol_result'])
+        pr = openfe.ProtocolResult.from_dict(d['protocol_result'])
 
-    assert pr
+        return pr
+
+    def test_reload_protocol_result(self, transformation_json):
+        d = json.loads(transformation_json,
+                       cls=gufe.tokenization.JSON_HANDLER.decoder)
+
+        pr = openmm_rfe.RelativeHybridTopologyProtocolResult.from_dict(d['protocol_result'])
+
+        assert pr
+
+    def test_get_estimate(self, protocolresult):
+        est = protocolresult.get_estimate()
+
+        assert est
+        assert isinstance(est, unit.Quantity)
+        assert est.is_compatible_with(unit.kilojoule_per_mole)
```

Comment:
Could we assert the value for the estimate & uncertainty? (I'm assuming this should remain static unless something goes terribly wrong)